docs/design/iam_containerized.md

```
@@ -134,14 +158,11 @@ access_keys: {
   - Accounts: ARN should `aws:arn:<account_id>:root`
   - IAM User: ARN should `aws:arn:<account_id>:user/<path>:Username`
 
-
-
-## IAM Policy Validation
+## IAM Inline Policy Validation
 - You can find the design doc [here](./IamUserInlinePolicy.md).  
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)